src/SumoCoders/FrameworkCoreBundle/Controller/Tests/DatePickerController.php

```diff
@@ -30,15 +30,6 @@ public function indexAction()
                 )
             )
             ->add(
-                'date_example2',
```

Comment:
The example sequence should be refactored to not have gaps.

Reply:
fixed 89a0a2e